editor/backend/src/EditCommand.ts

```diff
@@ -0,0 +1,113 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+import { isElectronMain } from "@bentley/bentleyjs-core";
+import { CommandMethodProps, CommandResult, editCommandApi, PingResult, StartCommandProps } from "@bentley/imodeljs-editor-common";
+
+/** @public */
+export type EditCommandType = typeof EditCommand;
+
+/**
+ * An EditCommand that performs an editing action on the backend. EditCommands are usually paired with and driven by EditTools on the frontend.
+ * EditCommands have a *commandId* that uniquely identifies them, so they can be found via a lookup in the [[EditCommandAdmin]].
+ * Every time an EditCommand runs, a new instance of (a subclass of) this class is created
+ * @public
+ */
+export class EditCommand {
+  /** The unique string that identifies this EditCommand class. This must be overridden in every subclass. */
+  public static commandId = "";
+  public static version = "1.0.0";
+
+  public constructor(_arg?: any) { }
+  public get ctor() { return this.constructor as EditCommandType; }
+
+  public onStart(): CommandResult<any> { return {}; }

```

Comment:
Can you document what these do and when they are called? Difference between onCleanup and onFinish?